android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/ScopedSecureStoreModule.kt

```diff
@@ -9,8 +9,8 @@ import host.exp.exponent.Constants
 
 private const val SHARED_PREFERENCES_NAME = "SecureStore"
 
-class ScopedSecureStoreModule(private val scopedContext: ScopedContext) :
-  SecureStoreModule(if (Constants.isStandaloneApp()) scopedContext.baseContext else scopedContext) {
+class ScopedSecureStoreModule(private val scopedContext: ScopedContext) : SecureStoreModule() {
```

Comment:
```suggestion
class ScopedSecureStoreModule(scopedContext: ScopedContext) : SecureStoreModule() {
```
I think, it's not needed 🤔